File: ClientApp/src/app/models/feedback.ts
import { ResultResponse } from "./result-response";
import { Guid } from "guid-typescript";

export class Feedback {
  id?: string;
  surveyId?: number;
  questionSetId?: number;
  contactTypeId?: number;
  respondentId?: string;
  dateTimeCreated?: Date;
  timeCompleted?: Date;
  resultResponse?: ResultResponse[];

  constructor() { 
  this.id = undefined;
  this.surveyId = undefined;
  this.questionSetId = undefined;
  this.contactTypeId = undefined;
  this.respondentId = undefined;
  this.dateTimeCreated = undefined;
  this.timeCompleted = undefined;
  }
}

File: Models/Feedback.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace SurveyTool.Models
{
    public partial class Feedback
    {
        public Feedback()

        {
            ResultResponse = new HashSet<ResultResponse>();
        }

        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }
        public int SurveyId { get; set; }
        public int QuestionSetId { get; set; }
        public int ContactTypeId { get; set; }
        public Guid? RespondentId { get; set; }

        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public DateTime DateTimeCreated { get; set; }
        public DateTime? TimeCompleted { get; set; }

        public virtual Survey Survey { get; set; }
        public virtual ContactType ContactType { get; set; }
        public virtual QuestionSet QuestionSet { get; set; }
        public virtual Respondent Respondent { get; set; }
        public virtual ICollection<ResultResponse> ResultResponse { get; set; }
    }
}
